app/database_etl/tableau_data_connector/google_services_manager.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, sheet_name):
 
     def create_service(self):
         creds = None
-        if os.path.exists('tableau_data_connector/token.pickle'):
-            with open('tableau_data_connector/token.pickle', 'rb') as token:
+        if os.path.exists('app/database_etl/tableau_data_connector/credentials.json'):
```

Comment:
this checks if credentials.json exists and then uses token.pickle below is that expected?

Reply:
Nice Catch!

Reply:
[80ad6f4](https://github.com/serotracker/iit-backend/pull/327/commits/80ad6f490dfd5245982593a4578ca9273106bcdc)